lib/src/components/Slider/Range.tsx

```diff
@@ -0,0 +1,415 @@
+import type { ChangeEvent, KeyboardEvent } from 'react'
+import { forwardRef, useCallback, useEffect, useRef, useState } from 'react'
+
+import { Hint } from '@/components/Hint'
+import { InputText } from '@/components/InputText'
+import { Text } from '@/components/Text'
+import { classNames } from '@/utils'
+
+import SliderStyles from './slider.module.scss'
+import type { RangeProps } from './types'
+
+import { round } from './index'
+
+export const thumbWidth = 20
+
+const cx = classNames(SliderStyles)
+
+/**
+ * Ensure minimum of a given value against a value `toCompare` based on a step
+ */
+const ensureMin = ({
+  min,
+  step,
+  toCompare,
+  value,
+}: {
+  min: number
+  step: number
+  toCompare: number
+  value: number
+}) => {
+  let ensuredValue = Math.min(value, toCompare - 1 * step)
+  ensuredValue = Math.max(ensuredValue, min)
+  return round(ensuredValue, step)
+}
+
+/**
+ * Ensure maximum of a given value against a value `toCompare` based on a step
+ */
+const ensureMax = ({
+  max,
+  step,
+  toCompare,
+  value,
+}: {
+  max: number
+  step: number
+  toCompare: number
+  value: number
+}) => {
+  let ensuredValue = Math.max(value, toCompare + 1 * step)
+  ensuredValue = Math.min(ensuredValue, max)
+  return round(ensuredValue, step)
+}
+
+/**
+ * @name Slider.Range
+ */
+// BREAKING CHANGE: removed `borderSelectorColor` default value
```

Comment:
Could you add this to `welcome-ui/website/build-app/pages/foundations/upgrades/v9.mdx` please?